services/graphql-server/src/graphql/resolvers/platform/asset.js

```diff
@@ -14,4 +18,24 @@ module.exports = {
     alt: image => createAltFor(image),
     caption: image => createCaptionFor(image.caption),
   },
+
+  /**
+   *
+   */
+  Mutation: {
+    assetImageUpdate: async (_, { input }, { base4rest, basedb }, info) => {
+      validateRest(base4rest);
```

Comment:
Should require auth

Reply:
The auth requirement is done in the directive, doesn't need to do anything on the resolver side -- but I did forget to make the definition use the directive, so i will update that ;P